src/utilities/game_launcher.py

```diff
@@ -48,8 +48,19 @@ def launch_runelite_with_settings(bot, settings_file: Path):
     key = bot.game_title.lower()
     EXECPATH = data.get(key, "")
 
+    if not os.path.exists(EXECPATH) and sys.platform == 'darwin':
```

Comment:
I'm sure `subprocess.Popen()` would be able to execute and run the RuneLite game. Is the issue that it is not appearing in Finder? I believe this can be solved by adding the `.app` extension to the list of searchable files in the `__locate_executable()` function below.

My Mac is kind of reserved for work so I don't know if I want to tinker with RS stuff, but maybe you could give that a shot?